oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoVersionGCSupport.java

```diff
@@ -100,6 +105,11 @@ public MongoVersionGCSupport(MongoDocumentStore store) {
         }
     }
 
+    /** accessor for testing only */
+    protected RevisionVector getLastDefaultNoBranchDeletionRevs() {
```

Comment:
I cannot find any usages of this method. Remove?

Reply:
agree, a left over from a previous test version. removed now